logging/logkey/constants.go

```diff
@@ -62,4 +62,19 @@ const (
 	// GitHubCommitID is the key used to represent the GitHub Commit ID where the
 	// Knative component was built from in logs
 	GitHubCommitID = "commit"
+
+	// AdmissionReviewUID is the key used to represent the admission review
+	// request/response UID in logs
+	AdmissionReviewUID = "knative.dev/admissionreview/uid"
```

Comment:
I don't think it's necessary to namespace these logging keys, but it doesn't especially hurt, either. They should compress fairly well.

Reply:
Agreed. Saving logging data is important for us.